doc/articles/features/uno-material.md

```diff
@@ -1,134 +1,4 @@
-`ToggleSwitch`|MaterialSecondaryToggleSwitchStyle<br>MaterialToggleSwitchStyle
-`winui:InfoBar`|MaterialInfoBarStyle
-`winui:ProgressBar`|MaterialProgressBarStyle<br>MaterialSecondaryProgressBarStyle
-`winui:ProgressRing`|MaterialProgressRingStyle<br>MaterialSecondaryProgressRingStyle
-
-### Styles for custom material controls
-Control|Resource Key
--|-
-`BottomNavigationBar`|MaterialBottomNavigationBarStyle
-`BottomNavigationBarItem`|MaterialBottomNavigationBarItemStyle
-`Card`|MaterialAvatarElevatedCardStyle<br>MaterialAvatarOutlinedCardStyle<br>MaterialBaseCardStyle<br>MaterialElevatedCardStyle<br>MaterialOutlinedCardStyle<br>MaterialSmallMediaElevatedCardStyle<br>MaterialSmallMediaOutlinedCardStyle
-`Chip`|MaterialFilledInputChipStyle<br>MaterialFilledChoiceChipStyle<br>MaterialFilledFilterChipStyle<br>MaterialFilledActionChipStyle<br>MaterialOutlinedInputChipStyle<br>MaterialOutlinedChoiceChipStyle<br>MaterialOutlinedFilterChipStyle<br>MaterialOutlinedActionChipStyle
-`ChipGroup`|MaterialFilledInputChipGroupStyle<br>MaterialFilledChoiceChipGroupStyle<br>MaterialFilledFilterChipGroupStyle<br>MaterialFilledActionChipGroupStyle<br>MaterialOutlinedInputChipGroupStyle<br>MaterialOutlinedChoiceChipGroupStyle<br>MaterialOutlinedFilterChipGroupStyle<br>MaterialOutlinedActionChipGroupStyle
-`Divider`|MaterialDividerStyle
-`ExpandingBottomSheet`|MaterialExpandingBottomSheetStyle
-`ModalStandardBottomSheet`|MaterialModalStandardBottomSheetStyle
-`SnackBar`|MaterialSnackBarStyle
-`StandardBottomSheet`|MaterialStandardBottomSheetStyle
-
-### Customizable Color Theme
-The colors used in the material styles are part of the color palette system, which can be customized to suit the theme of your application. Since this is decoupled from the styles, the application theme can be changed, without having to make a copy of every style and edit each of them.
-
-### Leading Icon for Button Control
-Many of the styles* above for the `Button` control support specifying an icon that is displayed adjacent to standard content:
-```xml
-xmlns:extensions="using:Uno.Material.Extensions"
-...
-<Button Content="DO WORK" Style="{StaticResource MaterialContainedButtonStyle}">
-   <extensions:ControlExtensions.Icon>
-      <FontIcon Glyph="&#xE9F5;" />
-   </extensions:ControlExtensions.Icon>
-</Button>
-```
-*Certain specialized `Button` types (ex: FAB, Pane) have styles which do not leverage this attached property because standard text content is not included
-
-## Additional Resources
-- Official Material Design site: https://material.io/design
-- Uno.Material library repository: https://github.com/unoplatform/Uno.Themes
-- [Additional control-specific setup](uno-material-controls-extra-setup.md)
-- Tools for picking colors: https://material.io/design/color/the-color-system.html#tools-for-picking-colors
+> [!IMPORTANT]
+> The documentation for Uno.Material has been moved to the [Material Getting Started](../external/uno.themes/doc/material-getting-started.md) page.
```

Comment:
https://platform.uno/docs/articles/external/uno.themes/doc/material-getting-started.html

This page is also using deprecated types. It suggests `<MaterialColors xmlns="using:Uno.Material" />` and `<MaterialResources xmlns="using:Uno.Material" />` which shows:

![image](https://user-images.githubusercontent.com/31348972/224555626-90789c33-c650-42e7-99e0-fbebeca9ef7c.png)

@kazo0 Can you update that documentation?

Reply:
@Youssef1313 yep the docs have been changed already, they just haven't been published yet

https://github.com/unoplatform/Uno.Themes/blob/master/doc/material-getting-started.md

Reply:
Great!

Reply:
This PR should get them updated the next time docs are pushed
https://github.com/unoplatform/uno/pull/11640